### DISCUSSION TITLE
Text Field in Template not multi line

### DISCUSSION THREAD

**Discussion by petegordon** [documenso/documenso#1411] [General]
I was looking to add a multi line text field to a  document template. I was trying to create a template that allowed for a field without the text being truncated.

![image](https://github.com/user-attachments/assets/2e210b42-afb7-4145-ab88-b858e110099a)

I think this is where the limitation is: https://github.com/documenso/documenso/blob/1a9dcadba5876e89fe2e684a1ccf1675fae40103/packages/lib/types/field-meta.ts#L42

That’s as far as I’ve gotten with it. May be an enhancement? Thanks for a super product being built in the open!

**Comment by catalinpit** [comment]
Hey Pete, are you looking for a field that supports multi-line text or to remove the truncation?

Also, thanks for the kind words. 🙏

**Reply by petegordon** [reply]
Looking for a field that supports multi-line text as input. Thanks!

**Reply by catalinpit** [reply]
> Looking for a field that supports multi-line text as input. Thanks!

Got it. I will bring this up internally. Thanks!

**Reply by cfredericksen** [reply]
This would be great to have. I want to pass a list to a template to generate a document but the signer only sees a truncated list when viewing the document. This would allow for adhoc contract building. The template contains static language and I could pass in dynamic language.
